packages/config/src/dynamo/dynamo.config.ts

```diff
@@ -19,6 +20,7 @@ export class ConfigProviderDynamo extends BasemapsConfigProvider {
   Style = new ConfigDynamoCached<ConfigVectorStyle>(this, ConfigPrefix.Style);
   TileSet = new ConfigDynamoBase<ConfigTileSet>(this, ConfigPrefix.TileSet);
   Provider = new ConfigDynamoCached<ConfigProvider>(this, ConfigPrefix.Provider);
+  Job = new ConfigDynamoCached<ConfigJob>(this, ConfigPrefix.Job);
```

Comment:
We use job as a processing job, this could get confusing as to what "job" are we talking about?

Any thoughts on other names?

Reply:
How about BatchJob? with prefix with `bt_`

Reply:
Some random thoughts
- ImageryImport `ii_`
- CogCreation `cc_`
- ImportJob `ij_`

Reply:
My preference is CogCreation, what do you think?